ansible/scripts/install_ansible.sh

```diff
@@ -22,9 +22,19 @@ base_dir=$( cd "$( dirname "$bin" )" && pwd )
 
 set -e
 
+# get the release id
+if [ -f /etc/os-release ]; then
```

Comment:
Would you happen to know if this file exists in Centos and redhat?

Reply:
I spun up a CentOS image and checked. This file exists in both CentOS and Red Hat